game/src/js/scenes/TitleScene.js

```diff
@@ -21,19 +21,20 @@ export default class TitleScene extends BaseScene {
   create() {
     let background = this.add.image(800, 330, 'title_background');
     this.cameras.main.startFollow(background);
-    
+    this.scene.manager.keys.GamepadScene.setVisible(false)
```

Comment:
It can get tedious to do this in every scene where we _don't_ want the gamepad. Maybe just requesting to enable it when we do need it would be better.

Reply:
Ah, that's a good point. We can hide it when we transition scenes on the `changeToScene` method of `BaseScene` so it happens in the background for every scene.

Reply:
oh that's a great place to put it, I'll move it there!